### ISSUE TITLE
Using 'ESC' key, Nav controls are not getting closed.

### ISSUE THREAD

**Issue by yangster23** [microsoft/fluentui#14577]
### Environment Information

- **Package version(s)**: 7.64.0
- **Browser and OS versions**: Chrome Version 84.0.4147.105 (Official Build) (64-bit)

### Describe the issue:

Using the Esc key to close modals and fly-outs, nav dropdowns is becoming so prevalent in user interfaces that it’s usually expected by users. This is the best practice for Accessibility. 

Using 'ESC' key nav controls is not getting closed. Issue is reproduced in UI fabric documentation - https://developer.microsoft.com/en-us/fluentui#/controls/web/nav

MAS 2.4.5 - Multiple Ways (26)

![image](https://user-images.githubusercontent.com/5368099/90454798-b5efc400-e0a0-11ea-8d31-329c8d237695.png)

#### Actual behavior:

Using 'ESC' key, nav controls is not getting closed.

#### Expected behavior:

Opened nav controls should be closed as soon as the 'ESC' key is pressed.

**Comment by khmakoto** [comment]
Hi @yangster23, do you mean that the `expanded` portions of the `Nav` should be collapsed when using `Esc` key? Or do you mean something else? If you do mean that, do you have any ARIA supporting documentation for this behavior? As far as I'm aware this is not something that is required/considered by the ARIA spec. Thanks!
